pkgs/by-name/Xr-Fragments/package.nix

```
@@ -0,0 +1,46 @@
+{
+  stdenv,
+  fetchgit,
+  haxe,
+  nodejs,
+  yarn,
+  makeWrapper,
+  lib,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xrfragment";
+  version = "0.5.2";
+
+  src = fetchgit {
+    url = "https://codeberg.org/coderofsalvation/xrfragment.git";
+    rev = "v0.5.2";
+    hash = "sha256-19zfnV7pVAAdft/KrXX4xbb3EgLw+M5SoEGXcKHR2kg=";
+  };
+
+  buildInputs = [
+    haxe
+    nodejs
+    yarn
+    makeWrapper
+  ];
+
+  buildPhase = ''
+    runHook preBuild
+
+    # build command
```

This was just an illustrative example, what I was trying to say is that you should insert the build commands here, not replace them with this comment ...

```suggestion
    The command that builds the program should go here
```